src/text-editor-component.js

```diff
@@ -2652,16 +2655,25 @@ module.exports = class TextEditorComponent {
         const textNodeEndColumn =
           textNodeStartColumn + textNode.textContent.length;
 
+        // set clientPixelPosition value --
         if (nextColumnToMeasure < textNodeEndColumn) {
           let clientPixelPosition;
           if (nextColumnToMeasure === textNodeStartColumn) {
-            clientPixelPosition = clientRectForRange(textNode, 0, 1).left;
+            clientPixelPosition = clientRectForRange(textNode, textNode.textContent.length-1, textNode.textContent.lengt).right;
```

Comment:
```suggestion
            clientPixelPosition = clientRectForRange(textNode, textNode.textContent.length-1, textNode.textContent.length).right;
```

Tests should be written for this